### PR TITLE
split modal and form components from params

### PR DESCRIPTION
**Pull request by nyatindopatrick**
Refactored the code for easy testing.

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -22,27 +29,27 @@ export default function SheetJSApp(props) {
...
-                .trim()
-                .replace(
-                  /UNKNOWN|Unknown|County|Sub County|Invalid code|[.,]+/g,
-                  ''
-                )
-            : ''
+              .capitalize()
+              .trim()
+              .replace(
+                /UNKNOWN|Unknown|County|Sub County|Invalid code|[.,]+/g,
```
Comment (moshonk) [inline src/components/index.js:49]: Please capitalize `'Invalid Code'` and include `'Invaild Category'` then call `.trim()` at the end to clear any resultant white spaces, as discussed earlier today.

File: src/components/index.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+/* eslint-disable no-plusplus */
+/* eslint-disable react/no-this-in-sfc */
+/* eslint-disable no-extend-native */
+/* eslint-disable import/no-extraneous-dependencies */
 import React, { useState, useEffect, useRef } from 'react';
 import XLSX from 'xlsx';
-import { Button, Modal, Form, Input } from 'antd';
+import {
+  Button, Modal, Form, Input,
+} from 'antd';
```
Comment (moshonk) [inline src/components/index.js:10]: Please make  this import statement a one-liner to make it consistent with the rest

File: src/helpers/transform.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+const isInvalid = (input) => input
+  && !/UNKNOWN|Unknown|NONE|N\/A|Invalid code|Invalid category.+/g.test(input);
+
+const replaceAddr = (addr, txt = '') => `${addr
+  .toString()
+  .replace(/UNKNOWN|Unknown|County| Sub County|Invalid code|[.,]+/g, '')
```
Comment (moshonk) [inline src/helpers/transform.js:6]: Please remove the leading whitespace before `Sub County`. It might mess up your string matching

Comment (moshonk) [reply src/helpers/transform.js:6]: You also need to call `.trim()` at the end to ensure that any resulting white spaces are removed

Comment (nyatindopatrick) [reply src/helpers/transform.js]: Removed

File: src/components/index.js (hunk 2 above)
Comment (nyatindopatrick) [reply src/components/index.js]: Done

**Pull request merged**